src/gui/layerGroup.js

```diff
@@ -343,6 +343,28 @@ export class LayerGroup {
     return this.#baseScale;
   }
 
+  /**
+   * Get the base layer.
+   *
+   * @returns {ViewLayer|undefined} The base layer.
+   */
+  getBaseLayer() {
```

Comment:
This method already exists as `getBaseViewLayer`, could you update it to do the same as what you are suggesting?